integration-tests/tests/docload/docload.spec.js

```diff
@@ -15,31 +15,35 @@ limitations under the License.
 */
 
 module.exports = {
-
+  '@tags': ['safari-10.1'],
```

Comment:
Does it do anything if you have explicit conditions for Safari everywhere?

Reply:
Some tests won't ever work for safari 10 eg lontask, webvitals etc. So idea is to tag everything that should theoretically work with this tag so we can easily run them

Reply:
Right, but you haven't modified any test runner methods, and I don't see the tag in use anywhere.